src/tools/gophish_import.py

```diff
@@ -305,7 +306,7 @@ def main() -> None:
         logging.shutdown()
 
     except Exception as e:
-        logging.debug(f"{type(e)}: {e}")
+        logging.debug("%s: %s", type(e), e)
```

Comment:
Why not use [`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.Logger.exception) here instead of recreating the wheel?

Reply:
Yeah its been changed to use exception. Makes more sense in this context. 

See 44e24edb9ac96eb42759bdea8a9229563811ca59

Reply:
Its also been updated here 3ee423033fefa9873fd13190c0e6edb575cd0080